connaisseur/tests/data/alerting/alertconfig.json

```diff
@@ -0,0 +1,75 @@
+{
+  "admit_request": {
+    "message": "CONNAISSEUR admitted a request",
+    "templates": [
+      {
+        "custom_headers": "{\"Authorization\": \"GenieKey 12345678-abcd-2222-3333-1234567890ef\"}",
```

Comment:
To simplify, you can make `custom_headers` a list of strings of the form `"Authorization: GenieKey..."`. That way, you save the escaping. It's safe because header names can't contain colons.

To recover the dict, you can do something like
```
for header in custom_headers:
    k, v = header.split(":", 1)  # Right-hand side returns a tuple, two-variable assignment unpacks it automatically
    k, v = k.strip(), v.strip()  # similar
```
and take it from there.
```